Roys/app/src/main/java/com/dwg_karrier/roys/ListActivity.java

```diff
@@ -0,0 +1,86 @@
+package com.dwg_karrier.roys;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.ListView;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by userpc on 2017-01-16.
+ */
+
+public class ListActivity extends AppCompatActivity{
+  ListView lv;
+  ArrayList<item> data;
+  Date fin_time; /* expected finish time */
+  Date cur_time; /* current time*/
+  long duration; /* time duration between current_time and finish time */
+
+  protected void onCreate (Bundle savedInstanceState){
+    super.onCreate(savedInstanceState);
+    setContentView(R.layout.list);
+
+    final Intent intent = new Intent(this.getIntent());
+    fin_time = (Date)intent.getSerializableExtra("fin_time");
+    cur_time = (Date)intent.getSerializableExtra("cur_time");
+
+    duration = (fin_time.getTime() - cur_time.getTime()) / 60000;
+    data = getData();
+
+    lv = (ListView)findViewById(R.id.listView);
+    lv.setAdapter(new ListViewAdapter(this, R.layout.item, data));
+
+    lv.setOnItemClickListener(new AdapterView.OnItemClickListener() {
+      @Override
+      public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
+        Intent intent1 = new Intent(ListActivity.this, ContentView.class);
+        intent1.putExtra("fin_time", fin_time);
+        intent1.putExtra("cur_time", cur_time);
+        startActivity(intent1);
+        finish();
+      }
+    });
+  }
+
+  private ArrayList<item> getData() {
+    ArrayList ret = new ArrayList<item>();
+
+    /*url list 불러오는 함수*/
+    //ret = callurl();
+
+    // imsi test code //
```

Comment:
We don't need // at the end of the comment: https://google.github.io/styleguide/javaguide.html#s4.8.6-comments
Recommend to use temp or some english words instead of imsi.